### ISSUE TITLE
Issue with HA 2023.8.0

### ISSUE THREAD

**Issue by FROCHE38** [fmartinou/tydom2mqtt#121]
Update today to HA 2023.8.0 and I have this warning
![image](https://github.com/fmartinou/tydom2mqtt/assets/37023141/2721a9b4-f99a-46ff-a761-c50be1a2bb6f)

**Comment by cob94440** [comment]
Same issue for me.
And since yesterday i have this :
```
--- Logging error ---
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/logging/__init__.py", line 1110, in emit
    msg = self.format(record)
          ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/logging/__init__.py", line 953, in format
    return fmt.format(record)
           ^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/logging/__init__.py", line 687, in format
    record.message = record.getMessage()
                     ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/logging/__init__.py", line 377, in getMessage
    msg = msg % self.args
          ~~~~^~~~~~~~~~~
TypeError: not all arguments converted during string formatting
Call stack:
  File "/app/main.py", line 125, in <module>
    main()
  File "/app/main.py", line 121, in main
    loop.run_forever()
  File "/usr/local/lib/python3.11/asyncio/base_events.py", line 607, in run_forever
    self._run_once()
  File "/usr/local/lib/python3.11/asyncio/base_events.py", line 1922, in _run_once
    handle._run()
  File "/usr/local/lib/python3.11/asyncio/events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "/app/main.py", line 55, in listen_tydom
    await message_handler.incoming_triage()
  File "/app/tydom/MessageHandler.py", line 309, in incoming_triage
    await self.parse_response(incoming)
  File "/app/tydom/MessageHandler.py", line 346, in parse_response
    logger.warning('Unknown message type received', data)
Message: 'Unknown message type received'
Arguments: ('[]\r\n',)
```
I'm on latest HA 2023.8.1 and latest tydom2mqtt.
I only have covers and kline door sensors.
This morning my covers weren't opened.

**Comment by FROCHE38** [comment]
Same issue in the log

**Comment by cob94440** [comment]
Hello,
This morning again my cover weren't open : 
in log : 


```
The above exception was the direct cause of the following exception:
Traceback (most recent call last):
  File "/app/mqtt/MqttClient.py", line 131, in on_message
    await Cover.put_positionCmd(tydom_client=self.tydom, device_id=device_id, cover_id=endpoint_id,
  File "/app/sensors/Cover.py", line 124, in put_positionCmd
    await tydom_client.put_devices_data(device_id, cover_id, 'positionCmd', positionCmd)
  File "/app/tydom/TydomClient.py", line 256, in put_devices_data
    await self.connection.send(a_bytes)
  File "/usr/local/lib/python3.11/site-packages/websockets/legacy/protocol.py", line 635, in send
    await self.ensure_open()
  File "/usr/local/lib/python3.11/site-packages/websockets/legacy/protocol.py", line 935, in ensure_open
    raise self.connection_closed_exc()
websockets.exceptions.ConnectionClosedError: no close frame received or sent
2023-08-07 07:10:00,077 - mqtt.MqttClient      - INFO    - set_positionCmd message received (topic=cover/tydom/1614676385_1614676385/set_positionCmd, message=UP)
2023-08-07 07:10:00,077 - sensors.Cover        - INFO    - 1614676385 positionCmd UP
2023-08-07 07:10:00,078 - asyncio              - ERROR   - Task exception was never retrieved
future: <Task finished name='Task-16903' coro=<MqttClient.on_message() done, defined at /app/mqtt/MqttClient.py:77> exception=ConnectionClosedError(None, None, None)>
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/websockets/legacy/protocol.py", line 959, in transfer_data
    message = await self.read_message()
              ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/websockets/legacy/protocol.py", line 1029, in read_message
    frame = await self.read_data_frame(max_size=self.max_size)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/websockets/legacy/protocol.py", line 1104, in read_data_frame
    frame = await self.read_frame(max_size)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/websockets/legacy/protocol.py", line 1161, in read_frame
    frame = await Frame.read(
            ^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/websockets/legacy/framing.py", line 68, in read
    data = await reader(2)
           ^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/asyncio/streams.py", line 729, in readexactly
    await self._wait_for_data('readexactly')
  File "/usr/local/lib/python3.11/asyncio/streams.py", line 522, in _wait_for_data
    await self._waiter
  File "/usr/local/lib/python3.11/asyncio/selector_events.py", line 970, in _read_ready__get_buffer
    nbytes = self._sock.recv_into(buf)
ConnectionResetError: [Errno 104] Connection reset by peer
The above exception was the direct cause of the following exception:
Traceback (most recent call last):
  File "/app/mqtt/MqttClient.py", line 131, in on_message
    await Cover.put_positionCmd(tydom_client=self.tydom, device_id=device_id, cover_id=endpoint_id,
  File "/app/sensors/Cover.py", line 124, in put_positionCmd
    await tydom_client.put_devices_data(device_id, cover_id, 'positionCmd', positionCmd)
  File "/app/tydom/TydomClient.py", line 256, in put_devices_data
    await self.connection.send(a_bytes)
  File "/usr/local/lib/python3.11/site-packages/websockets/legacy/protocol.py", line 635, in send
    await self.ensure_open()
  File "/usr/local/lib/python3.11/site-packages/websockets/legacy/protocol.py", line 935, in ensure_open
    raise self.connection_closed_exc()
websockets.exceptions.ConnectionClosedError: no close frame received or sent
2023-08-07 07:10:00,079 - mqtt.MqttClient      - INFO    - set_positionCmd message received (topic=cover/tydom/1612196495_1612196495/set_positionCmd, message=UP)
2023-08-07 07:10:00,079 - sensors.Cover        - INFO    - 1612196495 positionCmd UP
2023-08-07 07:10:00,079 - asyncio              - ERROR   - Task exception was never retrieved
future: <Task finished name='Task-16904' coro=<MqttClient.on_message() done, defined at /app/mqtt/MqttClient.py:77> exception=ConnectionClosedError(None, None, None)>
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/websockets/legacy/protocol.py", line 959, in transfer_data
    message = await self.read_message()
              ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/websockets/legacy/protocol.py", line 1029, in read_message
    frame = await self.read_data_frame(max_size=self.max_size)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/websockets/legacy/protocol.py", line 1104, in read_data_frame
    frame = await self.read_frame(max_size)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/websockets/legacy/protocol.py", line 1161, in read_frame
    frame = await Frame.read(
            ^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/websockets/legacy/framing.py", line 68, in read
    data = await reader(2)
           ^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/asyncio/streams.py", line 729, in readexactly
    await self._wait_for_data('readexactly')
  File "/usr/local/lib/python3.11/asyncio/streams.py", line 522, in _wait_for_data
    await self._waiter
  File "/usr/local/lib/python3.11/asyncio/selector_events.py", line 970, in _read_ready__get_buffer
    nbytes = self._sock.recv_into(buf)
             ^^^^^^^^^^^^^^^^^^^^^^^^^
ConnectionResetError: [Errno 104] Connection reset by peer
The above exception was the direct cause of the following exception:
Traceback (most recent call last):
  File "/app/mqtt/MqttClient.py", line 131, in on_message
    await Cover.put_positionCmd(tydom_client=self.tydom, device_id=device_id, cover_id=endpoint_id,
  File "/app/sensors/Cover.py", line 124, in put_positionCmd
    await tydom_client.put_devices_data(device_id, cover_id, 'positionCmd', positionCmd)
  File "/app/tydom/TydomClient.py", line 256, in put_devices_data
    await self.connection.send(a_bytes)
  File "/usr/local/lib/python3.11/site-packages/websockets/legacy/protocol.py", line 635, in send
    await self.ensure_open()
  File "/usr/local/lib/python3.11/site-packages/websockets/legacy/protocol.py", line 935, in ensure_open
    raise self.connection_closed_exc()
websockets.exceptions.ConnectionClosedError: no close frame received or sent
```

**Comment by fmartinou** [comment]
`HA 2023.8 mqtt entity naming convention warnings` and `Temperature and alarm attributes are not updated` fixed with `3.3.1`
